web/authenticate/models.py

```diff
@@ -1,5 +1,8 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 
+from happenings.models import Schedule
+
+
 class User(AbstractUser):
-    pass
+    swiped_events = models.ManyToManyField(Schedule)
```

Comment:
Could you rename it to interested_events since swiped_events would also imply all the events but it is only including the events swiped right

Reply:
Thats very much true! Will change now!